.github/workflows/ruby_test.yml

```diff
@@ -138,7 +145,7 @@ jobs:
     runs-on: ubuntu-latest
     env:
       RAILS_ENV: cucumber
-      CUCUMBER_FORMAT: summary
+      CUCUMBER_FORMAT: pretty
     # Services
```

Comment:
Travis used to complain if we used the pretty formatter, as it is very verbose. Re-enabled it to help diagnose the failures. Github Actions doesn't complain.